modules/rostests/rosautotest/main.cpp

```diff
@@ -113,7 +113,7 @@ wmain(int argc, wchar_t* argv[])
         ss << endl
            << endl
            << "[ROSAUTOTEST] System uptime " << setprecision(2) << fixed;
-        ss << ((float)GetTickCount()/1000) << " seconds" << endl;
+        ss << ((float)TestStartTime / 1000) << " seconds" << endl;
```

Comment:
Maybe remove the extra parentheses.

And line 166 should be
`ss << "[ROSAUTOTEST] Duration was " << (float)(TestEndTime - TestStartTime) / (60 * 1000);`

Reply:
> Maybe remove the extra parentheses.

Ok, and on line 159 too.

Reply:
Changed. Thanks.